### PR TITLE
Updated Pie charts example datasets to maintain consistency

### LINKED ISSUE #212: Update Pie Chart example datasets and reflow logic to maintain consistency
# Story
JIRA: [UXPLATFORM-5472](https://jira2.cerner.com/browse/UXPLATFORM-5472)
## Description

As a developer, I would like to make changes to the example datasets for pie charts.

## Motivation

So that consistency can be maintained across all the examples present in the terra-dev site.

## Definition of Done

We'll know we're done when the datasets and reflow logic for pie chart examples are consistent:

- All datasets are updated to be non-array.	
- Dedicated graph config objects are added for the examples where they are dynamically updated.

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
### Summary

- Updated datasets in pie chart examples to maintain consistency with other examples.

Closes #212 

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/terra-graphs-docs/src/example-datasets/dataObjects/Pie/simplePieDataset1.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import { loadPiePopup } from '@cerner/terra-graphs-docs/lib/example-datasets/popup';
+
+const dataset1 = {
+  key: 'uid_1',
```
Comment (sdadn) [inline packages/terra-graphs-docs/src/example-datasets/dataObjects/Pie/simplePieDataset1.js:5]: nit: update key to reflect dataset e.g. `simple_pie_data_1`

Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/example-datasets/dataObjects/Pie/simplePieDataset1.js]: Fixed [here](https://github.com/cerner/terra-graphs/pull/217/commits/62aee1949e9eb7274d2b3127fb2e2c1db836613a#diff-5c503e90dfa8f2b1ca5ccef303a90ef06d5d50cbfdc2bbc555e6330e183ded09R5).

**Review by tbiethman — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Comment by sdadn** [comment]
Can you add `* Updated Pie chart snapshots.` in the `changed` section of `terra-graphs/CHANGELOG.md`

**Comment by AshishMotanamGurunadham** [comment]
> Can you add `* Updated Pie chart snapshots.` in the `changed` section of `terra-graphs/CHANGELOG.md`
Changed [here](https://github.com/cerner/terra-graphs/pull/217/files#diff-d24190ff244a9f2416eeb3cbe315e41345dab81295d0fcccd748cd7f6c93a84aR26).

File: packages/terra-graphs/CHANGELOG.md [hunk 2]
```diff
@@ -23,5 +23,5 @@
 * Changed
   * Made changes to WDIO test url's according to terra rules. 
   * Updated components to maintain consistent design.
-
+  * Updated Pie chart snapshots.
```
Comment (sdadn) [inline packages/terra-graphs/CHANGELOG.md:26]: nit: newline after this

Comment (AshishMotanamGurunadham) [reply packages/terra-graphs/CHANGELOG.md:26]: Fixed [here](https://github.com/cerner/terra-graphs/pull/217/files#diff-d24190ff244a9f2416eeb3cbe315e41345dab81295d0fcccd748cd7f6c93a84aR27).

**Review by sdadn — APPROVED** [review]

**Pull request merged**